.oelint-ruleset.json

```diff
@@ -0,0 +1,249 @@
+  "oelint.var.filesoverride": "warning",
+  "oelint.vars.homepageprefix": "warning",
+  "oelint.vars.homepageping": "warning",
+  "oelint.vars.insaneskip": "info",
+  "oelint.var.licenseremotefile": "info",
+  "oelint.vars.licensesdpx": "warning",
+  "oelint.vars.licfileprefix": "warning",
+  "oelint.vars.mispell": "warning",
+  "oelint.vars.multilineident": "info",
+  "oelint.vars.overrideappend": "warning",
+  "oelint.vars.pbpusage": "error",
+  "oelint.vars.dusageinpkgfuncs": "error",
+  "oelint.vars.pkgspecific": "error",
+  "oelint.vars.pkgspecific.RDEPENDS": "error",
+  "oelint.vars.pkgspecific.RRECOMMENDS": "error",
+  "oelint.vars.pkgspecific.RSUGGESTS": "error",
+  "oelint.vars.pkgspecific.RCONFLICTS": "error",
+  "oelint.vars.pkgspecific.RPROVIDES": "error",
+  "oelint.vars.pkgspecific.RREPLACES": "error",
+  "oelint.vars.pkgspecific.FILES": "error",
+  "oelint.vars.pkgspecific.pkg_preinst": "error",
+  "oelint.vars.pkgspecific.pkg_postinst": "error",
+  "oelint.vars.pkgspecific.pkg_prerm": "error",
+  "oelint.vars.pkgspecific.pkg_postrm": "error",
+  "oelint.vars.pkgspecific.ALLOW_EMPTY": "error",
+  "oelint.vars.pnbpnusage": "error",
+  "oelint.vars.pnusagediscouraged": "warning",
+  "oelint.vars.pythonpnusage": "info",
+  "oelint.vars.valuequoted": "error",
+  "oelint.var.rootfspostcmd": "warning",
+  "oelint.vars.spacesassignment": "warning",
+  "oelint.vars.specific": "error",
+  "oelint.vars.srcurioptions": "warning",
+  "oelint.vars.srcuriappend": "error",
+  "oelint.vars.srcurichecksum": "error",
+  "oelint.vars.srcuridomains": "info",
+  "oelint.vars.srcurifile": "warning",
+  "oelint.vars.srcurigittag": "warning",
+  "oelint.vars.srcurisrcrevtag": "error",
+  "oelint.var.srcuriwildcard": "error",
+  "oelint.vars.summary80chars": "info",
+  "oelint.vars.summarylinebreaks": "warning",
+  "oelint.vars.notrailingslash": "error",
+  "oelint.vars.downloadfilename": "warning",
+  "oelint.vars.filessetting": "warning",
+  "oelint.vars.filessetting.hidden": "warning",
+  "oelint.vars.filessetting.double": "warning",
+  "oelint.var.improperinherit": "error",
+  "oelint.var.inherit": "warning",
+  "oelint.var.inherit.inherit": "warning",
+  "oelint.var.inherit.inherit_defer": "warning",
+  "oelint.var.inheritdevtool": "warning",
+  "oelint.var.inheritdevtool.native": "warning",
+  "oelint.var.inheritdevtool.nativesdk": "warning",
+  "oelint.var.inheritdevtool.cross": "warning",
+  "oelint.vars.listappend": "error",
+  "oelint.var.mandatoryvar": "error",
+  "oelint.var.mandatoryvar.SUMMARY": "error",
+  "oelint.var.mandatoryvar.DESCRIPTION": "info",
+  "oelint.var.mandatoryvar.HOMEPAGE": "error",
+  "oelint.var.mandatoryvar.LICENSE": "error",
+  "oelint.var.mandatoryvar.SRC_URI": "error",
+  "oelint.var.multiinclude": "warning",
+  "oelint.var.multiinherit": "warning",
+  "oelint.var.nativefilename": "warning",
+  "oelint.var.nativesdkfilename": "warning",
+  "oelint.var.order": "info",
+  "oelint.var.order.SUMMARY": "info",
```

Comment:
Is it really that bad to enable it? Why not follow the rules of style guide from the upstream project: https://docs.yoctoproject.org/dev/contributor-guide/recipe-style-guide.html#recipe-ordering
IMHO, we should skip them only when there is a good reason to.

Avoiding following the rules, will just make the possible upstreaming unnecessarily more difficult task, so we will do it even lass then we do today.

Reply:
Done https://github.com/Dasharo/meta-dts/pull/129/commits/80542fdfc24abbcd050ea3e3ead7f7522d866b90#diff-cceeba45d98af3b36ef0a37e8ffdda5ff5d1ba0084c4ad21dad516a28f2e20a4R201-R228